tests/oshiftroutetests/oshift_route_passthrough_test.go

```diff
@@ -401,6 +401,8 @@ func TestPassthroughRouteWithAlternateBackends(t *testing.T) {
 	integrationtest.CreateSVC(t, "default", "absvc2", corev1.ServiceTypeClusterIP, false)
 	integrationtest.CreateEP(t, "default", "absvc2", false, false, "3.3.3")
 
+	time.Sleep(2 * time.Second) //service should be created before route
```

Comment:
here also, we have to check the shared informer cache and then proceed with the route creation.